src/app/services/api/medco-node/explore-cohorts.service.ts

```diff
@@ -36,7 +33,7 @@ export class ExploreCohortsService {
 
   getCohortSingleNode(node: ApiNodeMetadata): Observable<ApiCohortResponse[]> {
     return this.apiEndpointService.getCall(
-      'node/explore/cohorts',
+      'node/explore/cohorts?limit=0',
       false,
       node.url
     );
```

Comment:
> I tried using HttpParams structure, but it didn't seem to work. Maybe you know how to use it properly ?

Shouldn't the following work?

```suggestion
    const params = new HttpParams().set('limit', '0');
    return this.apiEndpointService.getCall(
      'node/explore/cohorts',
      {params},
      node.url
    );
```

Reply:
thank you, I'll try